dynamics_controllers/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>dynamics_controllers</name>
+  <version>0.0.0</version>
+  <description>Adds a "dynamics control layer" to standard effort controllers.</description>
+
+  <author email="franco.fusco@ls2n.fr">Franco FUSCO</author>
+  <maintainer email="franco.fusco@ls2n.fr">Franco FUSCO</maintainer>
```

Comment:
Could you please add my me as maintainer so I also get the build farm emails?

Reply:
Addressed in last commit